integreat_compass/cms/templates/_base.html

```diff
@@ -0,0 +1,33 @@
+{% extends "_raw.html" %}
+{% load i18n %}
+{% load static %}
+{% block raw_content %}
+    <div class="flex justify-between items-center shadow-md bg-white z-50 fixed w-screen top-0">
+        <div class="container mx-auto flex items-center font-medium text-xl text-gray-700 hover:text-gray-900">
```

Comment:
I see two issues here:
1. I think text-gray-700 is too dark. It translates to #374151 but the design suggests to use #757575. I see two options how to go about this: 
- Overwrite text-gray-700 in the config to #757575
- Use text-gray-500 instead, since it's closer to #757575

And we should already have in mind, that there will be paragraphs that are suggested to have #595959 as font color.

2. Due to the fact that the hover-effect is on the div that's wrapping all links, we have the behavior, that when you hover over one link all links get highlighted. I don't think this is intended. Instead I think the intended behavior is that only the link that gets hovered on is highlighted.

Reply:
> I see two issues here:
> 
>     1. I think text-gray-700 is too dark. It translates to #374151 but the design suggests to use #757575. I see two options how to go about this:
> 
> 
>     * Overwrite text-gray-700 in the config to #757575
> 
>     * Use text-gray-500 instead, since it's closer to #757575
> 

Yep, I'll switch it out for 500. Are we intending to match all the colors (esp. greys) exactly? I find some of them a little lacking in contrast tbh.

> And we should already have in mind, that there will be paragraphs that are suggested to have #595959 as font color.
> 
>     2. Due to the fact that the hover-effect is on the div that's wrapping all links, we have the behavior, that when you hover over one link all links get highlighted. I don't think this is intended. Instead I think the intended behavior is that only the link that gets hovered on is highlighted.

Yep, my mistake, thanks for noticing :)